### PR TITLE
Proposal: Version 1.7.2 - More import support, removed languages dependences, and more

### PR DESCRIPTION
**Pull request by cdevroe**
This is our proposal for version 1.7.2 of Unmark. It is a significant update. It includes the following changes:
- Import support for all Netscape HTML services including Pinboard, Delicious, and others #168, #159, #112 
- Import support for Readability - slightly improved from 1.7.1.1
- Removes gettext dependences (including ngettext and _() functions) #138 
- Removes all previous Locales
- base_url config variable is auto-configured #171 
- Removes all compiled JS/CSS files from Git
- Removes need for "ASSET_VERSION" variable in view_helper
- Less Grunt dependencies for building project
- Updated .gitignore
- Focuses form on login #118 
- Various other bug fixes and improvements #172, #136 

We hope to clean up this release and have it out this week. Please give it a spin on your local install. There are no migrations to run. Also, we are hard at work on Verison 1.8.0 which will introduce an updated design, better mobile support, and the foundation of new features which have been requested in the past.

IMPORTANT NOTE: In order to run this version  you must run grunt once you pull the source since the compressed JS / CSS files are no longer included in git

Thanks for everyone's help testing this build.

### CODE REVIEW

**Comment by cdevroe** [comment]
I'm going to clean up this branch a little bit and then prepare to release it today.

**Comment by jamesacampbell** [comment]
This looks good to me, eager to get pinboard importing working and I tried checking out this branch but the js and css break due to ver= path errors so please can we merge this with master and release 1.7.2? Thanks!

**Comment by cdevroe** [comment]
@m1guelpf @ysragh I am getting an upload error on importing marks. Can you verify on your end?

File: readme.md [hunk 1]
```diff
@@ -25,26 +25,46 @@ The local-version of Unmark is recommended for at least intermediate users so th
...
+
+#### From git repository
+- Run `git clone https://github.com/plainmade/unmark.git` (Or, if you've forked the repo, use your URL)
 - Copy `/application/config/database-sample.php` to `/application/config/database.php` (leave `database-sample.php` where it is if you cloned the repo)
 - Create a database for Unmark to use in mySQL (may we recommend "unmark" as a database name?)
 - Fill in proper database credentials in `/application/config/database.php`
-- Optionally: Update your HOSTS file and create a virtual host for Unmark. We use "unmark.local" most of the time.
-- Point your browser to `/setup`
+- Optionally: Update your HOSTS file and create a virtual host for Unmark. We use "unmark.local" as an example.
+- Run `npm install` from the application's root directory
```
Comment (m1guelpf) [inline readme.md]: (or Yarn install)

File: readme.md [hunk 2]
```diff
@@ -25,26 +25,46 @@ The local-version of Unmark is recommended for at least intermediate users so th
...
+- Run `grunt` from the app's root directory [more info on Grunt](http://gruntjs.com/)
```
Comment (m1guelpf) [inline readme.md]: (puntuaction between directory and the more info)

File: readme.md [hunk 3]
```diff
@@ -64,6 +84,7 @@ Another way is to contribute your own code via Pull Requests. Here are some note
 - Create your own branch `git checkout -b your-branch-name`
 - Update your code and push those code changes back to your fork's branch `git push origin your-branch-name`
 - [Submit a Pull Request](https://github.com/plainmade/unmark/pulls) using that branch
+- And please accept our _thanks_!
```
Comment (m1guelpf) [inline readme.md]: (maybe an emoji here? :wink:)

**Comment by m1guelpf** [comment]
Error when trying a fresh install with this branch:
```
A DATABASE ERROR OCCURRED

Error Number: 1067

Invalid default value for 'datearchived'

ALTER TABLE `users_marks` ADD `datearchived` timestamp NOT NULL

Filename: /home/m1guelpiedrafita/www/unmark/migrations/002_archive_stats.php

Line Number: 14
```
PHP 7.1

**Comment by emkae** [comment]
@cdevroe What version of PHP are you using? Apache, or Nginx?

**Comment by cdevroe** [comment]
@walialu I rolled an update into this version for the issue you had above. Would you mind trying it?

**Comment by walialu** [comment]
@cdevroe Thanks a lot! I'll try this evening. I'm currently at work. But when I'm back home and have put my daughter to bed, I'll try and get back to you! Thanks a lot, Colin!

**Comment by cdevroe** [comment]
[unmark.zip](https://github.com/cdevroe/unmark/files/1310985/unmark.zip)

From 1.7.2 on Unmark releases will have their own binary created by the build routine. Attached is a version of that. If anyone can help to test we'd appreciate it.

**Pull request merged**